frontend/src/components/pages/consumers/Modals.tsx

```diff
@@ -264,36 +263,39 @@ export class EditOffsetsModal extends Component<{
                         </Text>
                         <Text display="inline-block" ml="auto" padding="0 1rem">{items.length} Partitions</Text>
                     </Flex>,
-                    description: <Table
-                        size="small"
-                        showSorterTooltip={false}
-                        pagination={{pageSize: 1000, position: ['none', 'none'] as any}}
-                        dataSource={items}
-                        rowKey={r => r.partitionId}
-                        rowClassName={r => (r.newOffset == null ? 'unchanged' : '')}
+                    description: <DataTable<GroupOffset>
+                        size="sm"
+                        showPagination
```

Comment:
I see previously we had a default page size of 1000, is there any way to provide a default apart from just `showPagination` boolean flag?

Reply:
10 is a default now, Do you suggest I should put back 1000?

Reply:
No it's fine, as long as arbitrary default is set, then I think it's fine. @rikimaru0345 ?